src/univariate/discrete/skellam.jl

```diff
@@ -79,9 +81,14 @@ function cf(d::Skellam, t::Real)
     μ1, μ2 = params(d)
     exp(μ1 * (cis(t) - 1) + μ2 * (cis(-t) - 1))
 end
+#based on SciPy: https://github.com/scipy/scipy/blob/v0.15.1/scipy/stats/_discrete_distns.py
```

Comment:
i'd be keen to see a better reference or derivation of this, if anyone knows of one.

Reply:
You can see Eqn. (5) in https://www.jstor.org/stable/2333532?seq=1#metadata_info_tab_contents
It relates the skellam pdf(difference of two independent poisson r.v's) with noncentral chi-square pdf.

Reply:
@simonbyrne Or maybe simply put x=2*lambda1 ; k=2*(x+1) ; lambda = 2*lambda2 in the PDF given here at https://en.wikipedia.org/wiki/Noncentral_chi-squared_distribution.
That will yield the Skellam PDF. 
The CDF also looks exactly similar with a change of 2*x instead of 2*(x+1) in the args which I believe might be done due to some precision issues/continuity correction

Reply:
@Sumegh-git you should put the whole reference and computation in docstring form:
```julia
"""
    cdf(::Skellam, x)

Add the link, add the explanation
"""
```

Reply:
Ok!